### PR TITLE
Add Rendevous Routing - Christian Decker

### PR DESCRIPTION
**Pull request by Caralie**

### CODE REVIEW

**Comment by Caralie** [comment]
@adamjonas I'm not sure I did this transition how I was supposed to...but this is one step "righter" than the last time.

File: chaincode-labs/chaincode-residency/2019-06-28-christian-decker-rendezvous-routing.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title:  Christian Decker - Rendezvous Routing (2019-06-28)
+transcript_by:
```
Comment (adamjonas) [inline chaincode-labs/chaincode-residency/2019-06-28-christian-decker-rendezvous-routing.md:3]: Name(s) of transcript author should be included

**Review by adamjonas — COMMENTED** [review]
Looks good!

File: chaincode-labs/chaincode-residency/2019-06-28-christian-decker-rendezvous-routing.md (hunk 1 above)
Comment (Caralie) [reply chaincode-labs/chaincode-residency/2019-06-28-christian-decker-rendezvous-routing.md]: thank you! I've updated

**Pull request merged**